modules/cluster-logging-deploy-cli.adoc

```diff
@@ -139,14 +139,14 @@ metadata:
   name: "elasticsearch-operator"
   namespace: "openshift-operators-redhat" <1>
 spec:
-  channel: "4.7" <2>
+  channel: "5.0" <2>
   installPlanApproval: "Automatic"
   source: "redhat-operators" <3>
   sourceNamespace: "openshift-marketplace"
   name: "elasticsearch-operator"
 ----
 <1> You must specify the `openshift-operators-redhat` Namespace.
-<2> Specify `4.7` as the channel.
+<2> Specify `5.0` as the channel.
```

Comment:
Do we also need to mention `stable` as an option here and what that actually means: latest 5.x released code?

Reply:
@jcantrill Replaced it with `Specify `5.0` or `stable` as the channel. `stable` is the latest 5.x released code.`